ai/integrations/llama_index/README.md

```diff
@@ -4,23 +4,89 @@ You can use functions defined within Unity Catalog (UC) directly as tools within
 
 ## Installation
 
-### From PyPI
+### Client Library
+
+To use this package with **Open Source Unity Catalog**, you will need to install:
 
 ```sh
-pip install unitycatalog-llamaindex
+pip install unitycatalog-ai[oss]
 ```
 
-### From source
+To use this package with **Databricks Unity Catalog**, you will need to install:
 
-To get started with the latest version, you can directly install this package from source via:
+```sh
+pip install unitycatalog-ai[databricks]
+```
+
+### Integration Library
+
+With the appropriate client installed for the AI functionality for Unity Catalog, you can then install the `LlamaIndex` integration library:
 
 ```sh
-pip install git+https://github.com/unitycatalog/unitycatalog.git#subdirectory=ai/integrations/llama_index
+pip install unitycatalog-llamaindex
```

Comment:
Can we release unitycatalog-ai first, then update pyproject.toml file `databricks` to depend on `unitycatalog-ai[databricks]>=0.1.0`, `oss` to depend on `unitycatalog-ai[oss]>=0.1.0`

Reply:
We're actually removing the 'oss' naming convention here. I'll update this PR to reflect the changes (default will be standard Unity Catalog (from this repo), with [databricks] being an option for managed UC only).